package.json

```diff
@@ -100,8 +101,7 @@
     "build:sass": {
       "command": "sass --style=compressed --load-path=node_modules --load-path=node_modules/sass-true/sass $(ls -d */ | grep -vE 'node_modules|catalog')",
       "files": [
-        "**/*.scss",
-        "!catalog"
```

Comment:
Why is this one necessary to remove?

Reply:
Because the catalog doesn't contain `scss` files **and** the root is back to `workspaces` therefore the `@material/web` package is symlinked in root `node_modules` which is a directory ignored by wireit by default.  

I could add it back if you want, but it works fine without.

Reply:
I think you should add it back in case we do add scss as it would likely be a sub-command inside the catalog dir

Reply:
Ok it makes sense. I reverted.